samples/file-encryptor/enclave/mbedtls_src/aes_context.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+#ifndef AES_CONTEXT_H
+#define AES_CONTEXT_H
```

Comment:
Why not `#pragma once` instead, which is support by clang, gcc, msvc, intel's c++ compiler, etc.?

Reply:
removed header aes_context.h